ee/get-support.md

```diff
@@ -45,6 +45,26 @@ This support dump only contains logs for the node where you're running the
 command. If your UCP is highly available, you should collect support dumps
 from all of the manager nodes.
 
+## Use the CLI to get a support dump (multiple nodes)
+
+To get the support dump from the CLI for multiple nodes, use SSH to log into a UCP manager node
+and run:
```

Comment:
Can you rewrite this, or move the command below this sentence? After reading this users will expect to see the snippet below.

Reply:
completed.